src/Services/NewsletterService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\NewsletterSubscriber;
+use Illuminate\Http\Response;
+
+class NewsletterService
+{
+    public function isSubscriber(NewsletterSubscriber $subscriber): Response
```

Comment:
I don't like services returning HTTP responses. It should be reusable for every entrypoint.